api/aggregated_stats.go

```diff
@@ -46,11 +47,22 @@ func AggregatedStatsHandler(w http.ResponseWriter, r *http.Request) {
 		}
 	}
 
+	var until int64
+	if untilStr == "" {
+		until = time.Now().Unix()
+	} else {
+		var err error
```

Comment:
Nit: it looks exactly the same as the code in line 42. You can consider extracting it as a separate function and reuse it. The same in `raw_stats.go`.